src/routes/console/changeOrganizationTierCloud.svelte

```diff
@@ -0,0 +1,131 @@
+<script lang="ts">
+    import { Wizard } from '$lib/layout';
+    import { sdk } from '$lib/stores/sdk';
+    import { createEventDispatcher, onDestroy } from 'svelte';
+    import { addNotification } from '$lib/stores/notifications';
+    import Step1 from './wizard/cloudOrganizationChangeTier/step1.svelte';
+    import Step2 from './wizard/cloudOrganizationChangeTier/step2.svelte';
+    import Step3 from './wizard/cloudOrganizationChangeTier/step3.svelte';
+    import Step4 from './wizard/cloudOrganizationChangeTier/step4.svelte';
+    import Step5 from './wizard/cloudOrganizationChangeTier/step5.svelte';
+    import {
+        changeOrganizationFinalAction,
+        changeOrganizationTier,
+        changeTierSteps,
+        isUpgrade
+    } from './wizard/cloudOrganizationChangeTier/store';
+    import { goto, invalidate } from '$app/navigation';
+    import { Dependencies } from '$lib/constants';
+    import { Submit, trackEvent, trackError } from '$lib/actions/analytics';
+    import { page } from '$app/stores';
+    import { organization } from '$lib/stores/organization';
+    import { wizard } from '$lib/stores/wizard';
+    import { tierToPlan } from '$lib/stores/billing';
+
+    const dispatch = createEventDispatcher();
+
+    async function onFinish() {
+        await invalidate(Dependencies.FUNCTIONS);
+    }
+
+    async function changeTier() {
+        try {
+            const org = await sdk.forConsole.billing.updatePlan(
+                $organization.$id,
+                $changeOrganizationTier.billingPlan,
+                $changeOrganizationTier.paymentMethodId
+            );
+
+            //Add budget
+            if ($changeOrganizationTier?.billingBudget) {
+                await sdk.forConsole.billing.updateBudget(
+                    org.$id,
+                    $changeOrganizationTier.billingBudget,
+                    [75]
+                );
+            }
+
+            //Add collaborators
+            if ($changeOrganizationTier?.collaborators?.length) {
+                $changeOrganizationTier.collaborators.forEach(async (collaborator) => {
+                    await sdk.forConsole.teams.createMembership(
+                        org.$id,
+                        ['collaborator'],
+                        `${$page.url.origin}/console/organization-${org.$id}`,
+                        collaborator
+                    );
+                });
+            }
+
+            await invalidate(Dependencies.ACCOUNT);
+            dispatch('created');
+            await goto(`/console/organization-${org.$id}`);
+            if ($isUpgrade) {
+                addNotification({
+                    type: 'success',
+                    isHtml: true,
+                    message: `
+                    <b>Your organization has been upgraded</b>
+                   make the most of your increased resource capacity and continue building great things with Appwrite.`
```

Comment:
Can we keep it to just "Your organization has been upgraded" ?

@heyCarla @ArmanNik This is too much to read for how long this stays on screen, and a bit tacky

Reply:
Fine by me.